### DISCUSSION TITLE
Ozone does not work with Ranger

### DISCUSSION THREAD

**Discussion by MerelyOneMax1986** [apache/ozone#7381] [General]
Hi all,

I'm trying to configure Ozone 1.4.0 HA Kerberos and Ranger, manually from the binaries. 
I'm facing an issue authorizing my users in Ozone due to the reason:

SSSD daemon failed while fetching those users from my domain

As to details, for Ranger UserSync I used backend **nss** with **enumerate=true** in **sssd.conf**. 

Could you please provide me with the working production solution? 

Any suggestion?

**Comment by arp7** [comment]
Hi @MerelyOneMax1986, you may want to check with the Apache Ranger community.

**Reply by MerelyOneMax1986** [reply]
Hi @arp7 , thanks for your piece of advice. Could you name the place where I can find Apache Ranger community?

**Comment by arp7** [comment]
You can try the project mailing lists: https://ranger.apache.org/mail-lists.html
